examples/reminders/ios/environments/cloud/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>942907959183-04s26r7johocivkl2l61crop1ml2dpiu.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.942907959183-04s26r7johocivkl2l61crop1ml2dpiu</string>
+	<key>API_KEY</key>
+	<string>AIzaSyCCqeYhV__1lyufMKlrWK_BDQOva9j0aPY</string>
```

Comment:
GoogleService-Info.plist should be treated as a credential file. Please do not push it to the git repository